nevergrad/optimization/base.py

```diff
@@ -246,6 +247,23 @@ def remove_all_callbacks(self) -> None:
         """
         self._callbacks = {}
 
+    def request(self, *args: Any, **kwargs: Any) -> None:
+        """Requests a new point to ask.
+        It will be ask at the next call (last in first out).
```

Comment:
I think lifo is better because some algorithms may (will) use this feature for initialization. With lifo you make sure your point is asked first.

Reply:
Hum that's a good point. 
Although I think that intuitively I will think that its the other way around when using it "manually".
Could we add a kwarg like `prepend=False` that optimizers will change to True internally when calling the method if they need it? (e.g. to emulate a lifo only for those who need it but fifo by default)

Reply:
problem with using a kwargs, is that it could interfere with the instrumentation kwargs, so not that nice (though it is not very likely). Do you think it is really important?

Reply:
Discussion summary: keep LIFO to be able to `request` and `ask` straight away